homeassistant/components/alarmdecoder/__init__.py

```diff
@@ -101,7 +101,7 @@ def handle_rel_message(sender, message):
         hass.helpers.dispatcher.dispatcher_send(SIGNAL_REL_MESSAGE, message)
 
     controller = False
```

Comment:
Looks like this can be removed.

Reply:
done in https://github.com/home-assistant/core/pull/40037/commits/fe09295895b26919401b9f02e67d9835e79bb68b